### PR TITLE
Csa 26 voltage monitoring update

### PR DESCRIPTION
**Pull request by GabrielPlante**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Documentation for user story CSA-26

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: pages/docs/castor/voltage-monitoring/algorithm.md [hunk 1]
```diff
@@ -1,20 +1,37 @@
...
+      - no applied remedial actions
+      - security status UNKNOWN
+  - Compute the voltages for all VoltageCnecs (computeVoltages method)
+  - Compare the voltages to their thresholds. For the VoltageCnecs that have a voltage overshoot, get the associated remedial actions (with
+    an OnVoltageConstraint usage rule containing the VoltageCnec and the state's instant)
+    - If the VoltageCnec has no associated RA, log a warning that the voltage constraint cannot be remedied, and move on to the
+      next VoltageCnec.
+    - For every RA:
+      - If the remedial action is not a topological action, do not apply it and log a warning. Then move on to the next RA.
+      - If the RA is a topological action, apply it on the network
```
Comment (pet-mit) [inline pages/docs/castor/voltage-monitoring/algorithm.md:25]:
```suggestion
      - If the remedial action is not a network action, do not apply it and log a warning. Then move on to the next RA.
      - If the RA is a network action, apply it on the network
```

File: pages/docs/castor/voltage-monitoring/result.md [hunk 2]
```diff
@@ -39,6 +40,42 @@ This information is not directly accessible in the JSON file.
...
+public Set<NetworkAction> getAppliedRas(State state)
+
+// get activated network action IDs for a given state ID
+public Set<String> getAppliedRas(String stateId)
+~~~
+{% endcapture %}
+{% capture t2_json %}
+Example:
+~~~json
+"applied-cras": [
```
Comment (pet-mit) [inline pages/docs/castor/voltage-monitoring/result.md:61]:
```suggestion
"applied-ras": [
```

**Review by pet-mit — CHANGES_REQUESTED** [review]

File: pages/docs/castor/voltage-monitoring/algorithm.md [hunk 3]
```diff
@@ -1,20 +1,37 @@
 Here is a detailed description of how the voltage monitoring algorithm operates:
-- Apply optimal preventive remedial actions from RaoResult on the network
 - From the CRAC, get the set of states on which VoltageCnecs exist
+- Apply optimal preventive remedial actions from RaoResult on the network
```
[the remark this answers is not in the document]
Comment (GabrielPlante) [reply pages/docs/castor/voltage-monitoring/algorithm.md:3]: I simply copy pasted and adapted the same algorithm as described in the user story, it was inverted there but I can revert those line

File: pages/docs/castor/voltage-monitoring/algorithm.md (hunk 1 above)
Comment (GabrielPlante) [reply pages/docs/castor/voltage-monitoring/algorithm.md]: Done !

File: pages/docs/castor/voltage-monitoring/result.md (hunk 2 above)
Comment (GabrielPlante) [reply pages/docs/castor/voltage-monitoring/result.md]: Done !

File: pages/docs/castor/voltage-monitoring/algorithm.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (pet-mit) [reply pages/docs/castor/voltage-monitoring/algorithm.md]: yes, revert this change please

[the remark this answers is not in the document]
Comment (GabrielPlante) [reply pages/docs/castor/voltage-monitoring/algorithm.md]: Done !

**Review by pet-mit — APPROVED** [review]

**Pull request merged**